addons/l10n_in/tests/test_credit_note_hsn.py

```diff
@@ -0,0 +1,58 @@
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests import tagged
+from odoo import fields, Command
+
+from dateutil.relativedelta import relativedelta
+
+
+@tagged('post_install', '-at_install', 'post_install_l10n')
+class TestCreditNoteHSN(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(chart_template_ref='in')
+        cls.product_line_vals_1 = {
+            'name': cls.product_a.name,
+            'product_id': cls.product_a.id,
+            'account_id': cls.product_a.property_account_expense_id.id,
+            'partner_id': cls.partner_a.id,
+            'product_uom_id': cls.product_a.uom_id.id,
+            'l10n_in_hsn_code': '1234',
+            'quantity': 1.0,
+            'price_unit': 800.0,
+            'price_total': 800.0,
+            'currency_id': cls.company_data['currency'].id,
+            'amount_currency': 800.0,
+            'debit': 800.0,
+            'credit': 0.0,
+        }
+        cls.company_data["company"].write({
+            "street": "Block no. 401",
+            "street2": "Street 2",
+            "city": "City 1",
+            "zip": "500001",
+            "state_id": cls.env.ref("base.state_in_ts").id,
+            "country_id": cls.env.ref("base.in").id,
+            "vat": "36AABCT1332L011",
+        })
```

Comment:
```suggestion
        cls.company_data["company"].write({
            "state_id": cls.env.ref("base.state_in_ts").id,
        })
```
Don't need all of those.